nupic/research/monitor_mixin/temporal_memory_monitor_mixin.py

```diff
@@ -322,18 +326,20 @@ def _mmComputeTransitionTraces(self):
   # ==============================
   # Overrides
   # ==============================
-
   def compute(self, activeColumns, sequenceLabel=None, **kwargs):
+    # Append last cycle's predictiveCells to *predicTEDCells* trace
     self._mmTraces["predictedCells"].data.append(self.predictiveCells)
 
     super(TemporalMemoryMonitorMixin, self).compute(activeColumns, **kwargs)
 
+    # Append last cycle's predictiveCells to *predicTIVECells* trace
     self._mmTraces["predictiveCells"].data.append(self.predictiveCells)
     self._mmTraces["activeColumns"].data.append(activeColumns)
-
+    self._mmTraces["activeCells"].data.append(self.activeCellsIndices())
```

Comment:
In fact, `correctlyPredictedCells` is already computed as a derived trace in `_mmComputeTransitionTraces` (it's called `predictedActiveCells`).